video/image_writer.c

```diff
@@ -52,6 +68,15 @@ const struct image_writer_opts image_writer_opts_defaults = {
     .webp_compression = 4,
     .jxl_distance = 1.0,
     .jxl_effort = 4,
+    .avif_encoder = "libaom-av1",
+    .avif_format = "yuv420p",
+    .avif_opts = (char*[]){
```

Comment:
const char *[]

Reply:
cannot be made const because mp_set_avdict() expects a non-const pointer

Reply:
ah, never mind then